src/constants.js

```diff
@@ -61,3 +75,55 @@ export const TASKS_TABLE_DEFAULTS = {
     },
   },
 };
+
+const TASKS_PAGE_POPOVER_HEADER = <div>About tasks</div>;
+const TASKS_PAGE_POPOVER_BODY = (
+  <div>
+    Tasks allows you to run resource-intensive additional troubleshooting on
+    your connected systems. Ansible Playbooks are written by Red Hat to do the
+    selected tasks.
+    <br /> <br />
+    Eligible systems include systems connected to cloud.redhat.com with rhc, or
```

Comment:
As far as I know cloud.redhat.com is changed to console.redhat.com. We need to update the line #86.